src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -151,3 +157,21 @@ class BlockingProcess(
     else
       args
 }
+
+object StreamGuardian {
+  def props(closeable: Closeable): Props =
+    Props(new StreamGuardian(closeable))
+}
+
+/**
+ * Responsible for closing a stream in order to free up any blocked
+ * threads that attempt to read from the stream.
+ */
+class StreamGuardian(closeable: Closeable) extends Actor {
+
+  override def receive =
+    Actor.emptyBehavior
+
+  override def postStop(): Unit =
+    closeable.close()
```

Comment:
Thanks to the `SupervisorStrategy.stoppingStrategy` we are using for `BlockingProcess`, we don't have to wait for the answer of the Akka team wrt my question on FlowDock what happens if an actor fails during `postStop` and the parent decides to `Restart` (which is the default).